### PR TITLE
Fix image slider showing enlarged images

### LINKED ISSUE #9882: [BOOM-3916] Image slider showing enlarged images
> This issue has been migrated from the Forge. Read the [original ticket here](http://forge.prestashop.com/browse/BOOM-3916).

- _**Reporter:**_ ankur112
- _**Created at:**_ Sat, 30 Sep 2017 13:30:33 +0200

<p>Images get zoomed in the image slider</p>
        
* How to reproduce the issue ?

N/A

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.6.x
| Description?  | Reproduce fix from PR https://github.com/PrestaShop/PrestaShop/pull/10492 which source branch was deleted -> Fix/improvement for #9882 + some responsive optimization if module is enabled for mobile
| Type?         | bug fix
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #9882
| How to test?  | See #9882

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/13072)

### CODE REVIEW

File: themes/classic/_dev/css/components/imageslider.scss [hunk 1]
```diff
@@ -22,6 +22,28 @@
...
+        }
+        @media (max-width: 480px){
+          font-size: 1.5rem;
+        }
+      }
+      h3{
+        @media (max-width: 767px) {
+          font-size: 1rem;
+        }
+        @media (max-width: 480px){
```
Comment (eternoendless) [inline themes/classic/_dev/css/components/imageslider.scss:44]:
```suggestion
        @include media-breakpoint-down(sm) {
```

File: themes/classic/_dev/css/components/imageslider.scss [hunk 2]
```diff
@@ -22,6 +22,28 @@
...
+      .display-1{
+        @media (max-width: 767px) {
+          font-size: 1.875rem;
+        }
+        @media (max-width: 480px){
+          font-size: 1.5rem;
+        }
+      }
+      h3{
+        @media (max-width: 767px) {
```
Comment (eternoendless) [inline themes/classic/_dev/css/components/imageslider.scss:41]:
```suggestion
        @include media-breakpoint-down(md) {
```

File: themes/classic/_dev/css/components/imageslider.scss [hunk 3]
```diff
@@ -22,6 +22,28 @@
...
+        }
+        @media (max-width: 480px){
+          display:none;
+        }
+      }
+      .display-1{
+        @media (max-width: 767px) {
+          font-size: 1.875rem;
+        }
+        @media (max-width: 480px){
```
Comment (eternoendless) [inline themes/classic/_dev/css/components/imageslider.scss:36]:
```suggestion
        @include media-breakpoint-down(sm) {
```

File: themes/classic/_dev/css/components/imageslider.scss [hunk 4]
```diff
@@ -22,6 +22,28 @@
...
         color: white;
+        @media (max-width: 767px) {
+          font-size: .875rem;
+        }
+        @media (max-width: 480px){
+          display:none;
+        }
+      }
+      .display-1{
+        @media (max-width: 767px) {
```
Comment (eternoendless) [inline themes/classic/_dev/css/components/imageslider.scss:33]:
```suggestion
        @include media-breakpoint-down(md) {
```

File: themes/classic/_dev/css/components/imageslider.scss [hunk 5]
```diff
@@ -22,6 +22,28 @@
       max-width: 340px;
       .caption-description p {
         color: white;
+        @media (max-width: 767px) {
+          font-size: .875rem;
+        }
+        @media (max-width: 480px){
```
Comment (eternoendless) [inline themes/classic/_dev/css/components/imageslider.scss:28]:
```suggestion
        @include media-breakpoint-down(sm) {
```

But watch out, `sm` is 576px

File: themes/classic/_dev/css/components/imageslider.scss [hunk 6]
```diff
@@ -22,6 +22,28 @@
       max-width: 340px;
       .caption-description p {
         color: white;
+        @media (max-width: 767px) {
```
Comment (eternoendless) [inline themes/classic/_dev/css/components/imageslider.scss:25]: You should use bootstrap's includes:
```suggestion
        @include media-breakpoint-down(md) {
```

But watch out, `md` is 768px

File: themes/classic/_dev/css/components/imageslider.scss (hunk 1 above)
Comment (Quetzacoalt91) [reply themes/classic/_dev/css/components/imageslider.scss:44]: When looking at https://getbootstrap.com/docs/4.0/layout/overview/#responsive-breakpoints, isn't it supposed to be 

```
@include media-breakpoint-down(xs) {
```

?

File: themes/classic/_dev/css/components/imageslider.scss (hunk 2 above)
Comment (Quetzacoalt91) [reply themes/classic/_dev/css/components/imageslider.scss:41]: Did you mean `@include media-breakpoint-down(sm) {` ?

File: themes/classic/_dev/css/components/imageslider.scss (hunk 6 above)
Comment (Quetzacoalt91) [reply themes/classic/_dev/css/components/imageslider.scss:25]: Further is this document we can read:

```
@include media-breakpoint-down(md) {
  .carousel {
    .carousel-item {

[...]

      .caption-description {
        display: none;
      }
    }
  }
}
```

Modifying its size is a non sense as it is anyway hidden.

File: themes/classic/_dev/css/components/imageslider.scss (hunk 5 above)
Comment (Quetzacoalt91) [reply themes/classic/_dev/css/components/imageslider.scss:28]: Further is this document we can read:

```
@include media-breakpoint-down(md) {
  .carousel {
    .carousel-item {

[...]

      .caption-description {
        display: none;
      }
    }
  }
}
```

No need to redeclare this, it is already hidden.

**Comment by Quetzacoalt91** [comment]
ping @eternoendless

**Comment by Quetzacoalt91** [comment]
@eternoendless, there are questions waiting for an answer from you.

**Comment by Quetzacoalt91** [comment]
Well, regarding the original issue, it seems this PR does not fixes it.
I wasn't able to see the actual changes excepted the arrows moving a little bit in the div.

Without any progress on this PR, let's close it and try with another solution.

Regards

**Pull request closed without merging**